_episodes/identifiers.md

```diff
@@ -1,22 +1,85 @@
 ---
 title: "Persistent identifiers"
-teaching: 0
-exercises: 0
+teaching: 10
+exercises: 5
 questions:
-- "Key question (FIXME)"
+- "What is a persistent identifier?"
+- "Which data need a PID?"
+- "How do you get a PID?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Obtain a PID for your data"
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "Promote reproducability by assigning persistent identifiers to your processed data."
+- "Promote reuse by adding a persistent identifiers to your raw data."
+- "If your data do not have a PID, they will not be FAIR!"
 ---
 
+What is a persistent identifier?
+--------------------------------
 
-General info on DOIs and citing data (example ERA interim paper no. Citations; Dee et al)
+Persistent identifiers (PIDs) are required to address the *F* in FAIR. They are used to identify digital resources using a unique and standardized label. PIDs are persistent, meaning that we will be able to find the data long into the future, even if the actual location (e.g. URL) or name of the resource changes. Well known PIDs are the [digital object identifiers](https://en.wikipedia.org/wiki/Digital_object_identifier) (DOI) and the [Handle System](https://en.wikipedia.org/wiki/Handle_System). In fact, a DOI is a type of Handle System handle.
 
-CMIP5 and CMIP6 (lessons learned; version/concept licence)
+Assigning a PID to your data makes sense. It will make the data easier to find, promotes reproducibility and reuse, makes it possible to track the impact, and enables long-term availability of the data. Remember that we said that data should be *fair* to computers? By assigning PIDs to our datasets (and publications), large networks of linked data and information can be created automatically.
 
-Other (MIP) projects
+> ## Anatomy of a DOI
+> A DOI takes the form of a string with two parts, a prefix and a suffix, divided by a slash. Usually, a prefix corresponds to the publisher, and the suffix to the resource.
+> ![Anatomy of a DOI](../fig/doi_anatomy.png)
+{: .callout}
+
+> ## Exercise
+> - Where does the above linked DOI point to?
+> - Do you think this DOI will still be valid in 50 years?
+> - Who is responsible to update the DOI if you get a dead link?
+> - What do you think should happen if the journal ceases to exist?
+{: .challenge}
+
+Which data need a PID?
+----------------------
+
+In principle, all data (publications, software, observations, model output, figures, etc.) need a PID, but practically, this can be broken down into two types of data:
+- Raw data
+- Processed data
```

Comment:
```suggestion
Ideally, all research objects (publications, software, observations, model output, figures, etc.) have PIDs, so that the entire workflow can be traced through the links between them. Additionally, it often makes sense to make separate entries for raw and processed data.
```